modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerConfigurationChangesTest.java

```diff
@@ -172,18 +175,20 @@ public void setUp() throws Exception {
 
     @AfterEach
     public void tearDown() throws Exception {
-        distributionZoneManager.stop();
-        metaStorageManager.stop();
-        clusterCfgMgr.stop();
-        vaultMgr.stop();
-        generator.close();
+        IgniteUtils.closeAll(
```

Comment:
It seems that if we switch to `closeAllManually()`, most of the null-checks will become obsolete

Reply:
Since managers do not support `ManuallyCloseable`, I did not quite understand your proposal.

Reply:
Oops :) Sorry for that